src/Contrib.KafkaFlow.ProcessManagers.SqlServer/SqlServerProcessManagersStore.cs

```diff
@@ -0,0 +1,101 @@
+﻿using Dapper;
+using KafkaFlow.SqlServer;
+using Microsoft.Extensions.Options;
+using System.Data.SqlClient;
+using System.Text.Json;
+
+namespace KafkaFlow.ProcessManagers.SqlServer;
+
+public sealed class SqlServerProcessManagersStore : IProcessStateStore
+{
+    private readonly SqlServerBackendOptions _options;
+
+    public SqlServerProcessManagersStore(IOptions<SqlServerBackendOptions> options)
+        => _options = options?.Value ?? throw new ArgumentNullException(nameof(options));
+
+    public async ValueTask Persist(Type processType, Guid processId, VersionedState state)
+    {
+        var sql = """
+            MERGE INTO [processes] as [target]
```

Comment:
Can you add the schema here too so it becomes `MERGE INTO [process_managers].[processes]...`

Reply:
Hmm, not sure if that's an oversight; I was convinced I already did that but I've been shuffling some commits around and rebasing stuff so it might have gotten lost when fixing merge conflicts. Or I simply overlooked it. I'll fix it.